hack/terraform-quickstart/main.tf

```diff
@@ -25,6 +25,17 @@ resource "aws_instance" "worker_node" {
   }
 }
 
+resource "aws_instance" "master_node" {
+  ami           = "${data.aws_ami.coreos_ami.image_id}"
+  instance_type = "m3.medium"
+  key_name      = "${var.ssh_key}"
+  count         = "${var.additional_masters}"
```

Comment:
We could be more symmetrical and use `num_masters` and `num_workers` it'd just take adding a little logic to terraform to figure out that one of the `num_masters` is actually the bootstrap node and is special. So I think we'd want to validate that this number is 1 or greater or something like that in terraform.

I labeled it as `addtional_masters` to imply it doesn't include the bootstrap node because it was easier to write that way.